python/ray/services.py

```diff
@@ -89,18 +90,24 @@ def kill_process(p):
   if RUN_PHOTON_PROFILER or RUN_PLASMA_MANAGER_PROFILER or RUN_PLASMA_STORE_PROFILER:
     os.kill(p.pid, signal.SIGINT) # Give process signal to write profiler data.
     time.sleep(0.1) # Wait for profiling data to be written.
-  p.kill()
-  # Sleeping for 0 should yield the core and allow the killed process to process
-  # its pending signals.
-  time.sleep(0)
+
+  # Allow the process one second to exit gracefully.
+  p.terminate()
```

Comment:
Why reverse the order of `kill` and `terminate`? Doesn't it make sense to start with `kill`?